buildSrc/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    `kotlin-dsl` version "2.3.3"
```

Comment:
so it doesn't work with 2.3.3?

Reply:
No, Gradle can't apply our precompiled script plugin `kotlin-library` with 2.3.x versions.